program/src/loader_v4.rs

```diff
@@ -1,14 +1,14 @@
-#[deprecated(since = "2.2.0", note = "Use solana-loader-v4-interface instead")]
-pub use solana_loader_v4_interface::{
-    instruction::{
-        create_buffer, deploy, deploy_from_source, finalize, is_deploy_instruction,
-        is_finalize_instruction, is_retract_instruction, is_set_program_length_instruction,
-        is_set_program_length_instruction as is_truncate_instruction,
-        is_transfer_authority_instruction, is_write_instruction, retract,
-        set_program_length as truncate, set_program_length as truncate_uninitialized,
-        set_program_length, transfer_authority, write,
+#![deprecated(since = "2.3.0", note = "Use solana-loader-v4-interface instead")]
```

Comment:
The deprecated note on `pub use` would still not be "loud" right?

Reply:
yeah if we want these warnings raised, we actually need to reverse our current path deprecation policy. leave symbols scheduled to move in place with deprecation warning, `pub use` them at the new, suggested location, then move the symbols with removal of the deprecation warning

Reply:
Note the exclamation point -- this goes on the whole module, and is the same as if I had put `#[deprecated(...)]` on `pub mod loader_v4`

Reply:
ah conversation view trimmed context strikes again!

Reply:
> Note the exclamation point -- this goes on the whole module, and is the same as if I had put #[deprecated(...)] on pub mod loader_v4

Nice, perfect!